.gitignore

```diff
@@ -3,4 +3,5 @@
 .idea
 node_modules
 public
-yarn-error.log
\ No newline at end of file
+yarn-error.log
+.vscode
```

Comment:
Why is this changed?

Reply:
My bad. I was experimenting with somethings, and I changed this `.gitignore`.

I'll change it back to its default